holidays/countries/vanuatu.py

```diff
@@ -36,47 +55,49 @@ def _populate(self, year):
         super()._populate(year)
 
         # New Years Day.
-        self._add_new_years_day("New Year's Day")
+        self._add_observed(self._add_new_years_day("New Year's Day"))
 
-        if year >= 1991:
-            # Father Lini Day
-            self._add_holiday("Father Lini Day", FEB, 21)
+        if year >= 1999:
+            # Father Lini Day.
+            self._add_observed(self._add_holiday_feb_21("Father Lini Day"))
 
-        # Custom Chief's Day
-        self._add_holiday("Custom Chief's Day", MAR, 5)
+        # Custom Chief's Day.
+        self._add_observed(self._add_holiday_mar_5("Custom Chief's Day"))
 
-        # Good Friday
+        # Good Friday.
         self._add_good_friday("Good Friday")
 
-        # Easter Monday
+        # Easter Monday.
         self._add_easter_monday("Easter Monday")
 
-        # Labour Day
-        self._add_labor_day("Labour Day")
+        # Labour Day.
+        self._add_observed(self._add_labor_day("Labour Day"))
 
-        # Ascension Day
+        # Ascension Day.
         self._add_ascension_thursday("Ascension Day")
 
-        # Children's Day
-        self._add_holiday("Children's Day", JUL, 24)
+        # Children's Day.
+        self._add_observed(self._add_holiday_jul_24("Children's Day"))
 
-        # Independence Day
-        self._add_holiday("Independence Day", JUL, 30)
+        # Independence Day.
+        self._add_observed(self._add_holiday_jul_30("Independence Day"))
 
-        # Assumption Day
-        self._add_assumption_of_mary_day("Assumption Day")
+        # Assumption Day.
+        self._add_observed(self._add_assumption_of_mary_day("Assumption Day"))
 
-        # Constitution Day
-        self._add_holiday("Constitution Day", OCT, 5)
+        # Constitution Day.
+        self._add_observed(self._add_holiday_oct_5("Constitution Day"))
 
-        # National Unity Day
-        self._add_holiday("National Unity Day", NOV, 29)
+        # Unity Day.
+        self._add_observed(self._add_holiday_nov_29("Unity Day"))
 
-        # Christmas Day
+        # Christmas Day.
         self._add_christmas_day("Christmas Day")
 
-        # Family day
-        self._add_christmas_day_two("Family Day")
+        # Family Day.
+        name = "Family Day"
+        if self.observed and self._is_monday(dec_26 := self._add_christmas_day_two(name)):
```

Comment:
`self._add_christmas_day_two(name)` gets executed with observed=True only. Is that the intended behavior?

Reply:
No, it wasn't. Fixed, thank you!